grails-app/assets/javascripts/streama/directives/streama-video-player-directive.js

```diff
@@ -330,6 +330,10 @@ angular.module('streama').directive('streamaVideoPlayer', [
 						}
 						$scope.videoDuration = video.duration;
 						video.currentTime = $scope.options.customStartingTime || 0;
+            //reset playtime when near the end of file
+            if($scope.options.outro_start ? $scope.options.outro_start < video.currentTime : video.duration - video.currentTime < 120) {
+              video.currentTime = 0
```

Comment:
wait a second, this will literally reset the playback, correct? so a user will actually all of a sudden be transported to the beginning of the video. I believe in the issue you linked, the desire is simply that the continue_watching backend logic is reset, so that on next-time viewing it starts at the beginning, not right away in the middle of watching...

Reply:
Da error con los link externos solo mp4 funciona 
Tambien se deberia agregar el registro de los usuarios para evitar que sean admin y puedan borrar editar etc
Como tambien activar un periodo de prueba de 30 dias y pasado dicho tiempo tengan que pagar la membrecia por paypal o tarjetas

Reply:
As per google translate: 
>Error with external links only mp4 works
>You should also add the user registry to avoid being admin and can delete edit etc.
>As well as activate a trial period of 30 days and after that time have to pay the membership by paypal or cards

Reply:
Digo si se puediera